src/hooks/use-snapshots.tsx

```diff
@@ -39,6 +40,9 @@ interface SnapshotContextType {
 	isLoadingSnapshotUsage: boolean;
 	initiated: boolean;
 	snapshotCreationBlocked: boolean;
+	progress: number;
+	startProgress: () => void;
+	stopProgress: () => void;
```

Comment:
What do you think about being specific about creationProgress:
```suggestion
	progressValue: number;
	startCreationProgress: () => void;
	stopCreationProgress: () => void;
```

Reply:
Done https://github.com/Automattic/studio/pull/960/commits/02e0a4d67430e4cec523ccfda28a36907d1bf228